module-system/sov-modules-api/src/utils.rs

```diff
@@ -8,3 +10,8 @@ pub fn to_jsonrpsee_error_object(err: impl ToString, message: &str) -> ErrorObje
         Some(err.to_string()),
     )
 }
+
+pub fn generate_address<C: Context>(key: &str) -> <C as Spec>::Address {
```

Comment:
Wouldn't this function to be more suitable in new `data-generation` crate?

Reply:
The bank module uses this function and the data-generation crate depends on the bank module so we don't want to create circular dependencies